app/models/UserProfile.ts

```diff
@@ -15,6 +15,16 @@ export default class UserProfile extends BaseModel {
     @Column({ nullable: true })
     public dob: Date;
 
+    /**
```

Comment:
Either remove column comment or shorten it because it is too long (must be 120 characters).

Reply:
Any special reason why we are enforcing a 120 character limit?

Reply:
Mainly for consistency and to keep comments column descriptions short and to the point. Plus that is the rule we set for the eslint max-len. Please reference other models with comments for examples.

Longer comments such as this would be more appropriate in the commit/pull request description rather than 
the code itself.